pkgs/applications/misc/dupeguru/default.nix

```
@@ -14,6 +14,12 @@ python3Packages.buildPythonApplication rec {
     fetchSubmodules = true;
   };
 
+  patches = [(fetchpatch {
```

please add a comment as to when this can be removed.
```suggestion
  patches = [
    # merged to master, remove next version bump
    (fetchpatch {
```

---

OK, done